api/src/test/java/org/pmiops/workbench/db/dao/UserDaoTest.java

```diff
@@ -424,6 +433,68 @@ public void testFindUserByUsernames() {
         .containsExactly(alice, bob);
   }
 
+  @Test
+  public void
+      testFindUsersBetweenCreationTimeWithoutNewUserSurveyOrCode_returnsUsersCreatedBetweenTimes() {
+    Instant creationTimeWindowStart = Instant.parse("2000-01-01T00:00:00.00Z");
+    Instant creationTimeWindowEnd = creationTimeWindowStart.plus(1, ChronoUnit.DAYS);
+
+    // Invalid user near window start
+    userWithCreationTime(Timestamp.from(creationTimeWindowStart.minus(1, ChronoUnit.SECONDS)));
+    DbUser validUserNearWindowStart =
+        userWithCreationTime(Timestamp.from(creationTimeWindowStart.plus(1, ChronoUnit.SECONDS)));
+    DbUser validUserNearWindowEnd =
+        userWithCreationTime(Timestamp.from(creationTimeWindowEnd.minus(1, ChronoUnit.SECONDS)));
+    // Invalid user near window end
+    userWithCreationTime(Timestamp.from(creationTimeWindowEnd.plus(1, ChronoUnit.SECONDS)));
+
+    List<DbUser> users =
+        userDao.findUsersBetweenCreationTimeWithoutNewUserSurveyOrCode(
+            Timestamp.from(creationTimeWindowStart), Timestamp.from(creationTimeWindowEnd));
+    assertThat(users)
+        .containsExactlyElementsIn(
```

Comment:
nit, I don't think you need to build a list first then `containsExactlyElementsIn`, drop `ImmutableList.of` then use `containsExactly`

Reply:
Thanks, I missed this when looking through the assertion methods.